templates/testpmd/resource-specs/sriov-vm1-template.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: dpdk-vm-template
+objects:
+- apiVersion: kubevirt.io/v1
+  kind: VirtualMachine
+  metadata:
+    annotations:
+      openshift.io/display-name: "RHEL VM with SR-IOV-backed NIC and DPDK."
+      description: >-
+        Template for RHEL 8.4 VM, with a secondary interface backed by an SR-IOV node NIC.
+        Involves replacing the default NIC driver (iavf) to vfio-pci, to enable running DPDK.
+      openshift.io/documentation-url: "https://github.com/RHsyseng/cnv-supplemental-templates/tree/main/templates/testpmd"
+    name: ${VM_NAME}
+    namespace: ${WORKING_NAMESPACE}
+  spec:
+    dataVolumeTemplates:
+    - apiVersion: cdi.kubevirt.io/v1beta1
+      kind: DataVolume
+      metadata:
+        name: ${DV_NAME}
+      spec:
+        source:
+          registry:
+            pullMethod: node
+            url: ${IMAGE_URL}
+        storage:
+          resources:
+            requests:
+              storage: ${REQUESTED_DV_STORAGE_SIZE}
+    running: false
+    template:
+      spec:
+        domain:
+          cpu:
+            cores: ${{NUM_OF_CORES}}
+          devices:
+            disks:
+            - disk:
+                bus: virtio
+              name: ${DV_NAME}
+            - disk:
+                bus: virtio
+              name: cloudinitdisk
+            interfaces:
+            - masquerade: {}
+              name: default
+            - macAddress: ${SECONDARY_MAC}
+              name: sriov-test-network
+              sriov: {}
+            rng: {}
+          resources:
+            requests:
+              memory: ${MEMORY_REQUEST}
+        networks:
+        - name: default
+          pod: {}
+        - multus:
+            networkName: ${WORKING_NAMESPACE}/sriov-test-network
+          name: sriov-test-network
+        nodeSelector:
+          feature.node.kubernetes.io/network-sriov.capable: "true"
+        terminationGracePeriodSeconds: 30
+        volumes:
+        - dataVolume:
+            name: ${DV_NAME}
+          name: ${DV_NAME}
+        - cloudInitNoCloud:
+            networkData: |
+              ethernets:
+                '1':
+                  match:
+                    macaddress: ${SECONDARY_MAC}
+                  set-name: sriov1
+              version: 2
+            userData: |-
+              #cloud-config
+              chpasswd:
+                expire: false
+              password: password
+              runcmd:
+              - 'sudo sh /usr/sbin/change-nic-driver.sh'
+          name: cloudinitdisk
+parameters:
+- description: VM name
+  name: VM_NAME
+  value: sriov-vm1
+- description: Image (source container-disk) URL 
+  name: IMAGE_URL
+  value: docker://<URL of your image repository>
```

Comment:
this is not a valid value; if we do not provide a default valid source, this parameter should be mandatory (i.e without a default value)

Reply:
You are right, it's not a valid value. And if the user won't modify it - processing the template will fail, which makes this parameter mandatory.
I don't want to put a default value here (I can't put the one I am using, because it's internal), and this parameter shouldn't be randomly generated.

Reply:
`value` should just be dropped, making this param mandatory without a default value